content/en/docs/rate-limits.md

```diff
@@ -197,6 +197,60 @@ help you identify and resolve issues without consuming your production limits.
 
 We do **not** offer overrides for this limit.
 
+</div>
+<div class="boxed">
+
+## Consecutive Authorization Failures per Hostname per Account
+
+Similar to [Authorization Failures per Hostname per
+Account](#authorization-failures-per-hostname-per-account) but only applies to
+consecutive failures. This limit is designed to prevent clients from getting
+stuck forever in a loop of failed validations.
+
+### Limit
+
+Up to 3,600 consecutive authorization failures per hostname can be incurred by
+one account. The ability to incur authorization failures refills at a rate of 1
+per hostname every day and resets to zero if an authorization for that hostname
+is successfully validated. Once exceeded, the account is prevented from
+requesting new certificates for that hostname. Each time the subscriber attempts
+to request a certificate they will receive an error containing a link to our
+Self-Service Portal where they can unpause issuance for the paused hostname and
+up to 50,000 additional paused hostnames associated with their account.
```

Comment:
Maybe there is an off‐by‐one glitch here? If Boulder unpauses [up to 50,000 identifiers][BOULDER] per request, shouldn’t it be ‘the paused hostname and up to *49,999* additional paused hostnames’?

[BOULDER]: https://github.com/letsencrypt/boulder/blob/071b8c5b352c405bc5cb23d9cd0304afea9502ec/sa/sa.go#L1544-L1546

Reply:
Oh, fun! Yeah, I think you're correct here.